docs/modes/predict.md

```diff
@@ -138,7 +138,8 @@ The `Results` object contains the following components:
 
 - `Results.boxes`: `Boxes` object with properties and methods for manipulating bounding boxes
 - `Results.masks`: `Masks` object for indexing masks or getting segment coordinates
-- `Results.probs`: `torch.Tensor` containing class probabilities or logits
+- `Results.keypoints`: `Keypoints` object for with properties and methods for manipulating predicted keypoints.
+- `Results.probs`: `Probs` object for containing class probabilities or logits.
 - `Results.orig_img`: Original image loaded in memory
```

Comment:
@Laughing-q I think @glenn-jocher and I had a discussion about not using "logits" here as it refers to the raw output of the model.

Reply:
you mean removing the `or logits` part?

Reply:
done in https://github.com/ultralytics/ultralytics/pull/2977/commits/97f882e5a265260936db354c311523586e91d7c7